cap-primitives/src/rustix/fs/oflags.rs

```diff
@@ -13,16 +13,7 @@ pub(in super::super) fn compute_oflags(options: &OpenOptions) -> io::Result<OFla
         oflags |= OFlags::SYNC;
     }
     if options.dsync {
-        #[cfg(not(target_os = "freebsd"))]
```

Comment:
It would ignore it, and… well, useful but probably not worth trying to version-detect or anything for a release branch that should become EoL by the end of this year? (Or, this can wait, really not an important part of the PR for me)

Reply:
Yeah, it's an admittedly minor detail, but would you mind splitting this out into a separate PR?